### PR TITLE
fix: :fire: Allow automation to run from manual triggers.

### PR DESCRIPTION
**Pull request by whizzzkid**
Removing check `github.event_name == 'push'

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -96,7 +96,7 @@ jobs:
   release-pr:
     runs-on: ubuntu-latest
     needs: [test]
-    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    if: github.ref == 'refs/heads/main'
```
Comment (lidel) [inline .github/workflows/ci.yml:99]:
```suggestion
    if: github.ref == 'refs/heads/main' && (github.event_name == 'push' || github.event_name == 'workflow_dispatch')
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -108,7 +108,7 @@ jobs:
   release-assets:
     runs-on: ubuntu-latest
     needs: [test]
-    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+    if: contains(github.ref, 'refs/tags/')
```
Comment (lidel) [inline .github/workflows/ci.yml:111]:
```suggestion
    if: contains(github.ref, 'refs/tags/') && (github.event_name == 'push' || github.event_name == 'workflow_dispatch')
```

**Review by lidel — COMMENTED** [review]
adding `github.event_name == 'workflow_dispatch'` feels safer

**Review by SgtPooki — APPROVED** [review]
approving with lidel's suggestions. We shouldn't remove event type entirely.

**Review by lidel — APPROVED** [review]

**Comment by lidel** [comment]
@whizzzkid  feel free to merge when you have time to eyeball if it works as expected :)

**Pull request merged**